docs/source/integrating/version-5-auth.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Social Sign-in & Email Auth
+permalink: /v5/integration/auth/
+---
+
+## Authentication Strategies 
+
+There are several strategies to choose from when it comes to user authentication. Login methods can be enabled or disabled for both the **Coral Admin** and the **Comment Stream** separately thus allowing you to use different methods for different user groups. 
+
+You can also use "Allow Registration" to restrict or allow users that have not signed up before to register and create new user accounts. 
+
+To configure authentication for your instance of Coral, go to **Configure** > **Authentication**. 
+
+## Login with email authentication
+Best when you want Coral to manage all user registration, creation and authentication. Requires the least amount of setup and configuration. Users register by creating a Username (must be unique across users, no spaces or special chars), Email, and Password. All user details are stored in Coral's database. 
+
+By Default - users are sent a verification email and are prompted to verify email addresss, however completing the email verification step is not required to submit comments. 
```

Comment:
Not currently correct.

```suggestion

```

Reply:
Confirmed, verification emails are sent, but are not required.